### DISCUSSION TITLE
compute_hausdorff_distance returns units

### DISCUSSION THREAD

**Discussion by LReion** [Project-MONAI/MONAI#6864] [Q&A] [answered]
compute_hausdorff_distance calculates the unit returned by the Euclidean distance. If the spacing is not set, is the default spacing=(1.,1.,1.)? Is the unit of the final result in mm?

**Comment by KumoLiu** [comment, marked as answer]
Hi @LReion, thanks for your interest.
Yes, you are right, if `spacing` is None, spacing of unity is used for all images in the batch. And the actual distance measurements can vary based on your data.
https://github.com/Project-MONAI/MONAI/blob/faf6f74aa54486d110ac19192568bd957dcbcfef/monai/metrics/hausdorff_distance.py#L165-L171

Hope it helps, thanks!
